src/main/java/gregtech/common/blocks/BlockGregStairs.java

```diff
@@ -0,0 +1,14 @@
+package gregtech.common.blocks;
+
+import gregtech.api.GregTechAPI;
+import net.minecraft.block.BlockStairs;
+import net.minecraft.block.state.IBlockState;
+
+public class BlockGregStairs extends BlockStairs {
+
+    public BlockGregStairs(IBlockState state) {
+        super(state);
+        setCreativeTab(GregTechAPI.TAB_GREGTECH);
```

Comment:
Should this also set hardness, sounds, et. al.?

Reply:
afaik the superclass sets them